localization/polish/interfaces_l_polish.yml

```diff
@@ -3877,7 +3877,7 @@
  MARKET_PANEL_BALANCE_GOLD:0 "#v [FixedPointToInt(Goods.GetMarketBuyAndSellOrdersBalance)|o=D]#!"
  MARKET_PANEL_DETAILS_TAB_LABEL:1 "Zboží"
  MARKET_PANEL_DETAILS_TAB_LABEL_BOLD:0 "#bold $MARKET_PANEL_DETAILS_TAB_LABEL$#!"
- MARKET_PANEL_FOOD_SECURITY_TAB_LABEL:0 "Bezpečnost potravin"
+ MARKET_PANEL_FOOD_SECURITY_TAB_LABEL:0 "Potravinová bezpečnost"
```

Comment:
Je to bezpečnosť potravín v tom že sú dobre zakonzervované, alebo je to akože zabezpečenie potravín (dodávok)